robot/OutboundFundsNPSP/tests/create_contact.robot

```diff
@@ -41,5 +41,5 @@ Validate Contact
     Page Should Contain  ${first_name} ${last_name}
     # Validate via API
     &{contact} =     Salesforce Get  Contact  ${contact_id}
-    Should Be Equal  ${first_name}  &{contact}[FirstName]
-    Should Be Equal  ${last_name}  &{contact}[LastName]
+    Should Be Equal  ${first_name}  ${contact['FirstName']}
+    Should Be Equal  ${last_name}  ${contact['LastName']}
```

Comment:
This change fixes the error in Tests/Create Contact - `Value of variable '&{contact}[FirstName]' is not dictionary or dictionary-like.`

Reply:
Thank you@